CME/pages/CMEEvaluationPage.php

```diff
@@ -697,6 +699,32 @@ protected function getMessageSecondaryContent(SwatForm $form)
 	}
 
 	// }}}
+	// {{{ protected function sendCompletionEmail()
+
+	protected function sendCompletionEmail()
+	{
+		// only send email if quiz is complete
+		$account = $this->app->session->account;
+		if (!$this->credits->getFirst()->isEarned($account) ||
+			!$this->progress->quiz instanceof CMEQuiz) {
+			return;
+		}
```

Comment:
I wonder if we should loop through each credit here. It shouldn't be possible to have only complete some of the credits but it may help prevent bugs in the future.